src/lobby/controller.js

```diff
@@ -210,6 +203,10 @@ function LobbyController(CardshifterServerAPI, $scope, $timeout, $rootScope, $lo
     function formatTimeNumber(time) {
         return time < 10 ? "0" + time : time;
```

Comment:
Is it really worth adding a new framework/library just to add this feature? I'd prefer to reinvent the very simple wheel in that case.